props/saber_fett263_buttons.h

```diff
@@ -45,14 +78,56 @@ SaberFett263Buttons() : PropBase() {}
   bool swinging_ = false;
   void Loop() override {
     PropBase::Loop();
+    // Swing On gesture control this portion allows fine tuning of speed needed to ignite
+    if(!SaberBase::IsOn() ) {
+      if (millis() - saber_off_time_ < MOTION_TIMEOUT) {
+        SaberBase::RequestMotion();
+      // Edit '250' value in line below to change swing on sensitivity, 250 ~ 400 work best in testing
+        if (!swinging_ && fusor.swing_speed() > 250) {
+          swinging_ = true;
+          Event(BUTTON_NONE, EVENT_SWING);
+        }
+        if (swinging_ && fusor.swing_speed() < 100) {
+          swinging_ = false;
+        }
+      }
+    }
+	  
     if (!swinging_ && fusor.swing_speed() > 250) {
       swinging_ = true;
       Event(BUTTON_NONE, EVENT_SWING);
     }
+	  
+    if (auto_lockup_on_ && 
+    !swinging_ && 
+    fusor.swing_speed() > 120 && 
+    fusor.swing_speed() < 250 && 
```

Comment:
So, if I understand this correctly, this is intended to create a clash-like effect by turning on lockup for 200ms?
However, if the saber is held relatively still, we keep the lockup effect going until it moves quickly, right?
But why are we checking if swing_speed is less than 250? I mean, if the saber is moving fast we should end lockup, right?

Reply:
technically, clash is made up of BEGIN_LOCKUP and END_LOCKUP, the LOCKUP_DELAY is to allow time for the user to steady the blade, especially on harder clashes as you get a "bounce" on impact the would cause END_LOCKUP before you could steady, this varies a little by user and hilt I found so I wanted to let users increase if they are hitting harder or get more bounce in their impacts so they can still get lockup time to initiate.  The BEGIN_LOCKUP effect "covers up" for this time, I will be describing it in the write up, if you increase the delay you will want to also increase the time the BEGIN_LOCKUP effect appears to conceal the lockup from showing.
The less than 250 came from a compile error during testing, because we were checking for > 250 in an earlier if statement (I believe) it wasn't compiling until I put in the < 250.  I forgot specific error but I couldn't get past it initially and this addition fixed it.  The > 120 is what really controls it, I mean you can't go from 0 to 250 without becoming greater then 120.  The 120 is the result from a lot of trial and error to find the "Goldilocks" movements that were easy to perform but also definitive enough to tell the difference between moving the saber in a lockup with opponent and "pulling away" to end it, too low and lockup became too hard to initiate too high and the movement needed to end lockup became too exaggerated and difficult to perform, the combination was the result of a lot of testing and fine-tuning to get it "just right".

Reply:
please remove the fusor.swing_speed() < 250, it doesn't do anything and it's confusing to the reader